packages/beacon-node/src/api/impl/beacon/rewards/index.ts

```diff
@@ -9,5 +9,10 @@ export function getBeaconRewardsApi({chain}: Pick<ApiModules, "chain">): ServerA
       const data = await chain.getBlockRewards(block.message);
       return {data, executionOptimistic};
     },
+    async getSyncCommitteeRewards(blockId, validatorIds) {
```

Comment:
Just a note, if we switch to object based params, the name of parameter must match with the name defined in the API interface. Might be best to call it `validatorIds` everywhere as it is more precise.

Reply:
Good point thanks